File: dotnet/SDL.DXA.Modules.AlpacaForms/Models/FormResultLink.cs
using Sdl.Web.Common.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SDL.DXA.Modules.Forms.Models
{
    /// <summary>
    /// Form Result Link
    /// </summary>
    [SemanticEntity(EntityName = "FormResultLink", Vocab = CoreVocabulary, Prefix = "e")]
    public class FormResultLink : EntityModel
    {
        [SemanticProperty("e:status")]
        public string Status { get; set; }

        [SemanticProperty("e:link")]
        public Link Link { get; set; }
    }
}

File: java/dxa-alpaca-forms/alpaca-forms-dxa-module/src/main/java/com/sdl/dxa/modules/alpacaforms/model/FormResultLink.java
package com.sdl.dxa.modules.alpacaforms.model;

import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticEntity;
import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticProperty;
import com.sdl.webapp.common.api.model.entity.AbstractEntityModel;
import com.sdl.webapp.common.api.model.entity.Link;

import static com.sdl.webapp.common.api.mapping.semantic.config.SemanticVocabulary.SDL_CORE;

/**
 * Form Result Link
 *
 * @author nic
 */
@SemanticEntity(entityName = "FormResultLink", vocabulary = SDL_CORE, prefix = "e", public_ = false)
public class FormResultLink extends AbstractEntityModel {

    @SemanticProperty("e:status")
    private String status;

    @SemanticProperty("e:link")
    private Link link;

    public Link getLink() {
        return link;
    }

    public String getStatus() {
        return status;
    }
}
